src/Tracing/SamplingContext.php

```diff
@@ -48,4 +54,20 @@ public function setParentSampled(?bool $parentSampled): void
     {
         $this->parentSampled = $parentSampled;
     }
+
+    /**
+     * @param array<string, mixed>|null $additionalContext
```

Comment:
Can you please add a little description of what this function does, e.g. `Sets additional optional context data.`?